check/src/main/scala/rsc/checkbase/RscClassPath.scala

```diff
@@ -0,0 +1,27 @@
+package rsc.checkbase
+
+import cats.data._
+import com.monovore.decline._
+import java.io.File.pathSeparator
+import java.nio.file.{Files, Path, Paths}
+
+case class RscClassPath(paths: List[Path])
```

Comment:
This isn't specific to the Rsc portion of a check run, so maybe a different name? `CompileClasspath`?